### DISCUSSION TITLE
Changing Interface, compiling sources that implement the interface isn't correct

### DISCUSSION THREAD

**Discussion by ppyck** [vscode-abl/vscode-abl#246] [Q&A] [answered]
I've change an Interface, let's say ISynchronizeWidgetControl. In that interface I deleted a property cause I didn't want it public (earlier decision to make it public was wrong).
Now when opening a class that implemts the interface, delete the property in that class and save it, **the compiler gives me an error**:
`Interface '*.*.*.*.ISynchronizeWidgetControl' requires property or event 'EnteringControl' to be implemented by class '*.*.*.*..AblUiSecurityField'. (14755)`
This error isn't correct since I first deleted the proptery in the interface and saved the interface.
Build mode = Build everything and saving the Interface triggered build of all the classes implementing that interface. So I would think that afterwards deleting the property in an implementing class shouldn't return an error during compilation

**Comment by gquerret** [comment, marked as answer]
I'd say there's probably another instance of the rcode in your propath.
